cmd/statreceiver/entrypoint

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+set -euo pipefail
+
+exec ./statreceiver --input ./config.lua
```

Comment:
This could be all in the `ENTRYPOINT` of the `Dockerfile`, since you're not doing anything complicated here.